inference-engine/src/inference_engine/include/openvino/runtime/parameter.hpp

```diff
@@ -8,353 +8,127 @@
  */
 #pragma once
 
-#include <algorithm>
-#include <cctype>
-#include <iterator>
-#include <map>
-#include <memory>
-#include <string>
-#include <tuple>
-#include <typeinfo>
-#include <utility>
-#include <vector>
+#include <unordered_map>
 
-#include "openvino/core/except.hpp"
+#include "openvino/core/shared_any.hpp"
 #include "openvino/runtime/common.hpp"
 
 namespace ov {
 namespace runtime {
+class Core;
+class ExecutableNetwork;
+class RemoteContext;
+class RemoteTensor;
 
 /**
- * @brief This class represents an object to work with different parameters
- *
+ * @brief This class represents generic shared value object loaded from DLL
  */
 class OPENVINO_RUNTIME_API Parameter {
```

Comment:
Why do we need to have Parameter in new namespaces if we introduce sharedAny?

Reply:
An `ov::runtime::Paramter` is loaded from plugin SO. It is just a read-only wrapper over `ov::SharedAnyImpl` with `so` reference inside